__tests__/Itinerary.tests.js

```diff
@@ -0,0 +1,22 @@
+const Itinerary = require("../src/Itinerary");
+const Port = require("../src/Port");
+
+describe('Itinerary', () => {
+  let liverpool;
+  let newYork;
+  let itinerary;
+
+  beforeEach(() => {
+    liverpool = new Port('Liverpool');
+    newYork = new Port('New York'); 
+    itinerary = jest.fn();
+  });
+
+  it('can be instantiated', () => {
+    expect(new Itinerary()).toBeInstanceOf(Object);
+  })
+  it('can have ports', () => {
+    itinerary = new Itinerary([liverpool, newYork]);
+    expect(itinerary.ports).toEqual([liverpool, newYork]);
+  })
```

Comment:
One of the most useful ways to arrange the code is in the format Arrange - Act - Assert:

**Arrange** the data for the test,
**Act** on the method you want to test,
**Assert** the returns are what you expect them to be.
Following this layout will improve the readability of the tests and help keep the logic more focused:

```suggestion
  it('can have ports', () => {
    liverpool = new Port('Liverpool');
    newYork = new Port('New York');
    itinerary = new Itinerary([liverpool, newYork]);

    expect(itinerary.ports).toEqual([liverpool, newYork]);
  })
```